src/components/footer/footer.js

```diff
@@ -18,6 +18,9 @@ const branch = process.env.BRANCH ?? DEFAULT_BRANCH
 const chromaticUrl = `https://www.chromatic.com/library?appId=60d36a243a7c1f0049472423&branch=${branch}`
 const storybookUrl = `https://${branch}--60d36a243a7c1f0049472423.chromatic.com`
 
+// to temporary disable button and link on page
+const storybookChromaticEnabled = false
```

Comment:
Could you remove the chromatic.yml file as well while we disable this? Or is there an option for a GitHub action to temporarily disable an action? 🤔